src/main/java/net/coderbot/iris/mixin/rendertype/MixinRenderType.java

```diff
@@ -152,11 +152,7 @@ private static RenderType wrap(RenderType wrapped, GbufferProgram program) {
 	private static void iris$wrapEntityRenderTypes(ResourceLocation texture, CallbackInfoReturnable<RenderType> cir) {
 		RenderType base = cir.getReturnValue();
 
-		if (HandRenderer.isActive()) {
-			cir.setReturnValue(wrap(base, GbufferProgram.HAND));
-		} else {
-			cir.setReturnValue(wrap(base, GbufferProgram.ENTITIES));
-		}
+		cir.setReturnValue(wrap(base, GbufferPrograms.refine(GbufferProgram.ENTITIES)));
```

Comment:
This isn't needed - you can just specify ENTITIES, and GbufferPrograms will automatically handle it assuming the global state is correct at the time of actually drawing everything.

```suggestion
		cir.setReturnValue(wrap(base, GbufferProgram.ENTITIES));
```

Reply:
fixed.